### ISSUE TITLE
FR: Templater / JavaScript integration.

### ISSUE THREAD

**Issue by Kardzhilov** [logancyang/obsidian-copilot#391]
Is your feature request related to a problem? Please describe.
The Copilot plugin in Obsidian currently lacks the ability to generate content on automatically Note creation, integration with the Templater plugin would fix that.

Describe the solution you'd like
I propose integrating Copilot with Templater using javascript commands maybe? 

Describe alternatives you've considered
Manually copying AI-generated text into Templater-generated notes is possible but undermines workflow efficiency.
The next version of the meta-bind plugin will have improved JS integration so maybe you could trigger generations with in Note buttons in that case.

Additional context
My usecase for this would involve TTRPG where it would enable quick and structured content creation without navigating multiple interfaces, thus enhancing the storytelling experience. But i see it being benneficial for many other usecases.

**Comment by logancyang** [comment]
Thanks for reporting this, and sorry for the slow reply.

Copilot v3 is in code freeze, so we're closing this as not planned. v4 is a ground-up rewrite and is launching soon, with a very different architecture around agent mode.

If this is still an issue for you in v4 once it lands, please open a new issue and we'll take a fresh look. 🙏
